javascript/javascript3/week2/CurrencyCalculator/inde.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Try to use Prettier as a code formatted https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode